src/components/Pagination/Pagination.tsx

```diff
@@ -84,59 +85,59 @@ export const Pagination = ({
         ))}
       </select>
       {renderPaginationNumbers()}
-      <FirstPageIcon
-        tabIndex={currentPage !== 0 ? 0 : undefined}
-        className={cn(classes['pagination-icon'], {
-          [classes['pagination-icon--disabled']]: currentPage === 0,
-        })}
+      <button
+        className={cn(classes['icon-button'])}
         onClick={() => setCurrentPage(0)}
-        onKeyUp={(event) => {
-          if (event.key === 'Enter' || event.key === ' ') {
-            setCurrentPage(0);
-          }
-        }}
+        disabled={currentPage !== 0 ? false : true}
+        aria-label='Naviger til første side i tabell'
```

Comment:
We need to support multiple languages, and the design system should not have any hard-coded text (except in stories). Could you extract this to properties? You could possibly pass in all texts as one big object instead of having each as their own property.